eo-maven-plugin/src/main/java/org/eolang/maven/Transpiler.java

```diff
@@ -62,14 +69,34 @@ final class Transpiler {
      */
     private final Path pre;
 
+    /**
+     * For what severities transpiling must fail.
+     */
+    private final Set<ErrorSeverity> sevtofail;
+
     /**
      * Ctor.
      * @param tmp The temp
      * @param ppre The pre
+     * @param sevtofail For what sevtofail to fail
      */
-    Transpiler(final Path tmp, final Path ppre) {
+    Transpiler(final Path tmp, final Path ppre, final Set<ErrorSeverity> sevtofail) {
```

Comment:
@mximp it's still a compound name, you just don't use camel case (which makes it look uglier :) Just try to find _one_ noun, which will be descriptive enough.